packages/@aws-cdk/aws-ecs/test/cluster.test.ts

```diff
@@ -2140,6 +2140,21 @@ describe('cluster', () => {
 
 
   });
+
+  test('When importing ECS Cluster via Arn', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const clusterName = 'my-cluster';
+    const region = 'service-region';
+    const account = 'service-account';
+    const clusterArn = `arn:aws:ecs:${region}:${account}:service/${clusterName}`;
```

Comment:
This is not the Cluster ARN (it should be `arn:aws:ecs:<region>:<account>:cluster/<cluster-name>`).